pygithub/im_commits_pygithub.py

```diff
@@ -0,0 +1,66 @@
+"""Demo designed to find and store information about PyGithub."""
+from github import Github
+from pydriller import RepositoryMining, GitRepository
+from pydriller.domain.commit import ModificationType
+from fastapi import FastAPI
+
+test_dict = {}
+def output_hash_map(dictionary):
+    """Method to print out the hash map."""
+    # print headings
+    print("Branch", "\t Author", "\t Time", "\t Files", "\t Message")
+    # prints hashmap content
+    # for key in dictionary:
+
+
+    for key, _ in dictionary.items():
+        print(key)
+        for items in dictionary[key]:
+            for item in items:
+                print("\t" + str(item) + "\t")
+            # print("\t" + str(item))
+
+def get_repo_commits_py_github():
```

Comment:
I think it would be a better practice if this function is parametrized. For example, the `user_token` and the `repo` could both be the arguments passed into the function, so that the function is more general-purposed